BUILDING.md

```diff
@@ -107,6 +107,9 @@ make generate
 > * `no_cri`: A build tag disables building Kubernetes [CRI](http://blog.kubernetes.io/2016/12/container-runtime-interface-cri-in-kubernetes.html) support into containerd.
 > See [here](https://github.com/containerd/cri-containerd#build-tags) for build tags of CRI plugin.
 > * `no_devmapper`: A build tag disables building the device mapper snapshot driver.
+> * `seccomp`: Enables seccomp support in in the cri plugin
```

Comment:
double "in in" here; otherwise looks good

Reply:
Fixed.